setup.py

```diff
@@ -12,56 +12,61 @@
 #
 ##############################################################################
 
-import os
-
 from setuptools import setup, find_packages
 
-here = os.path.abspath(os.path.dirname(__file__))
 
-with open(os.path.join(here, 'README.rst')) as f:
-    README = f.read()
+def readfile(name):
+    with open(name) as f:
+        return f.read()
+
 
-with open(os.path.join(here, 'CHANGES.rst')) as f:
-    CHANGES = f.read()
+README = readfile("README.rst")
+CHANGES = readfile("CHANGES.rst")
 
 tests_require = [
-    'zope.testrunner',
+    "zope.testrunner",
 ]
 
-setup(name='repoze.sphinx.autointerface',
-      version='0.8.1.dev0',
-      description='Sphinx extension: auto-generates API docs '
-                  'from Zope interfaces',
-      long_description=README + '\n\n' +  CHANGES,
-      classifiers=[
+setup(
+    name="repoze.sphinx.autointerface",
+    version="0.8.1.dev0",
```

Comment:
Please change the version number to 1.0.0.dev0 here and in the change log - there's obvious backwards-incompatible changes in this package

Reply:
Done in [8e0176c](https://github.com/repoze/repoze.sphinx.autointerface/pull/17/commits/8e0176cc2a52b4208984f7e8fec87fe687e4f139)